samples/sensor/max31855/README.rst

```diff
@@ -0,0 +1,35 @@
+.. _max31855:
+
+Maxim MAX31855 Cold-Junction Compensated Thermocouple-to-Digital Converter
+##########################################################################
+
+Overview
+********
+
+This sample application demonstrates how to use the Maxim MAX31855
+cold-junction compensated thermocouple-to-digital converter to measure
+temperature from a thermocouple and the die temperature in °C.
```

Comment:
We are moving away from driver-specific sensor sample applications. Please extend `samples/sensor/die_temp_polling` instead of adding a new sample

Reply:
If I'm not mistaken, I don't see any changes needed to the `die_temp_polling` sample to get it to support the sensor, and the other sensor channel is covered by the `thermometer` sample. Does removing the driver-specific sample suffice?

Reply:
Yes, thank you